--- a/chat_template.jinja
+++ b/chat_template.jinja
@@ -107,7 +107,7 @@
     {%- if loop.index0 >= num_sys %}
     {%- set content = render_content(message.content, true)|trim %}
     {%- if message.role == "system" or message.role == "developer" %}
-        {{- raise_exception('System message must be at the beginning.') }}
+        {{- '<|im_start|>system\n' + content + '<|im_end|>\n' }}
     {%- elif message.role == "user" %}
         {{- '<|im_start|>' + message.role + '\n' + content + '<|im_end|>' + '\n' }}
     {%- elif message.role == "assistant" %}
